style.css

```diff
@@ -0,0 +1,253 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```

Comment:
Also in this document, please be mindful about the use of empty lines. They should only be used to indicate a division between two independent 'parts' (e.g. classes, IDs, HTML blocks, etc.).

Reply:
The empty lines exist for no particular reason. I'll be more careful to avoid leaving them.